<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>427</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="SearchLayout">
   <widget class="QTextError" name="Field_Output">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>661</width>
      <height>409</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>409</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_Work">
     <item>
      <layout class="QHBoxLayout" name="Layout_Find">
       <item>
        <widget class="QPushButton" name="Button_Search">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Find_Year">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Старше чем (год)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Layout_Add">
       <item>
        <layout class="QHBoxLayout" name="Add">
         <item>
          <widget class="QLineEdit" name="Add_Mark">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Марка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Add_Year">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Год</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Add_Color">
           <property name="currentText">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Цвет</string>
           </property>
           <item>
            <property name="text">
             <string comment="Цвет">Цвет</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Add_Button">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Layout_Maintenance">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Записать в файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PrintAll_Button">
         <property name="text">
          <string>Вывести БД</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Считать БД</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextError</class>
   <extends>QTextBrowser</extends>
   <header location="global">qtexterror.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Button_Search</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SearchPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Add_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PrintAll_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>PrintDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DumpDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChooseDBFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SearchPosition()</slot>
  <slot>AddPosition()</slot>
  <slot>PrintDB()</slot>
  <slot>DumpDB()</slot>
  <slot>ReadDB()</slot>
  <slot>ChooseDBFile()</slot>
 </slots>
</ui>
